web/firefoxcom.js

```diff
@@ -346,6 +358,10 @@ class FirefoxExternalServices extends DefaultExternalServices {
     return new MozL10n(mozL10n);
   }
 
+  static get scripting() {
+    return FirefoxScripting;
+  }
```

Comment:
Where's the corresponding definition in `DefaultExternalServices`, in `web/app.js`?